### DISCUSSION TITLE
Completions screen not showing (anymore) in PhpStorm

### DISCUSSION THREAD

**Discussion by parijke** [community/community#7719] [Copilot Conversations] [answered]
I dragged the **show completions** tab to my second screen. After that, it won't show the completions suggestions. The screen flickers when using the key-combi (Alt+\) but no extra suggestions.

The plugin does show a single suggestion in the code though

**Comment by jansorg** [comment]
Thanks for reporting!

Is the window of "Show completions" the split tab editor opened by "Open GitHub Copilot"? 
A screenshot would be great to understand what exactly isn't working with the plugin.

**Comment by parijke** [comment]
No, I had to assign another key to Open Github Project to show Copilot Solutions.

This is happening when using Alt+\
![Peek 2021-11-15 12-44](https://user-images.githubusercontent.com/7671998/141776846-ff012efd-5061-4787-9728-8ab791411e37.gif)

**Reply by jansorg** [reply]
I don't fully understand the problem yet.
Is this `ALT++` or perhaps `Alt+\`? The shortcut seems to be cut off due to markdown.

Could you post your keymap settings of `Preferences > Keymap > Plugins > GitHub Copilot`, please?
Here's an example:
![Screenshot_20211115_125206](https://user-images.githubusercontent.com/11473063/141777612-059c8527-78cf-40ac-a588-12ecb8c632ae.png)

**Comment by parijke** [comment]
![image](https://user-images.githubusercontent.com/7671998/141778176-a8c8474c-e831-46dd-8175-fafadb300ab8.png)

Alt+\ does flicker the screen but nothing happens. The previous gif shows the screen flicker when using Alt+\

**Reply by jansorg** [reply]
Thanks! 
What were you expecting instead of the suggestion in the editor?

`Alt+\` is bound to `Copilot: Show completions` which is fetching suggestions from the server and updating the editor with the first suggestion (that's the gray block between lines 72 and 73). I agree that the plural (`completions`) is a bit confusing.

"Open GitHub Copilot"  (bound to `Ctrl+Alt+NumPad *` in your settings) is showing up to 10 suggestions at once in a split tab. This tab window isn't supported on a different screen at this time. Did you try to do this?

Thanks for your feedback, this will help to discover how to make it easier and less confusing to use!

**Comment by parijke** [comment, marked as answer]
Yes, I tried to drag the suggestions to the second screen. Didn't work. I am able to get the suggestions using my custom key mapping.  So I guess it is now working as designed then.
Thanks

**Reply by jansorg** [reply]
Thanks! We're tracking this internally now for possible improvements.
Unfortunately, Drag&Drop of the split tab is currently broken.
